test/suites/rt-upgrade/test-upgrade-chain.ts

```diff
@@ -0,0 +1,63 @@
+import { describeSuite, beforeAll, expect } from "@moonwall/cli";
+import { alith } from "@moonwall/util";
+import { ApiPromise } from "@polkadot/api";
+import { parseEther } from "ethers";
+import "@moonbeam-network/api-augment";
+
+describeSuite({
+  id: "CHU",
+  title: "Chopsticks Upgrade",
+  foundationMethods: "chopsticks",
+  testCases: ({ it, context }) => {
+    let api: ApiPromise;
+    const DUMMY_ACCOUNT = "0x11d88f59425cbc1867883fcf93614bf70e87E854";
+
+    beforeAll(() => {
+      api = context.polkadotJs();
+    });
+
+    it({
+      id: "T1",
+      title: "Calling chain constants data",
+      test: async () => {
+        const specVersion = api.consts.system.version.specVersion.toNumber();
+        expect(specVersion).to.be.greaterThan(0);
+      },
+    });
+
+    it({
+      id: "T2",
+      title: "Do an upgrade test",
+      timeout: 30000,
+      test: async function () {
+        const rtBefore = api.consts.system.version.specVersion.toNumber();
+        await context.upgradeRuntime(context);
```

Comment:
This should be inside a before no ? cause otherwise if I run T3 only it won't test the runtime upgrade

Reply:
done